static/styles/app_components.css

```diff
@@ -111,6 +111,7 @@
         &:hover,
         &:focus {
             border-color: hsl(0, 0%, 60%);
+            text-decoration-line: none;
```

Comment:
This is a pretty general CSS change -- can you put it in its own commit?  

I'm a little surprised we don't have this already, but it appears the reason is that we usually do buttons with a separate `a` element, rather than doing an `a` element with a bunch of button styles.

It's worth checking if the variant approach taken here might have other side effects, and/or if we have other existing buttons like this using `sea-green`.  Grepping, I see we indeed had 2 cases of this, plus a normla link that incorrectly has the `sea-green` class on it.

```
$ git grep sea-green | grep '<a'
static/templates/settings/account_settings.hbs:                                    <a href="/accounts/password/reset/" class="sea-green" target="_blank" rel="noopener noreferrer">{{t "Forgotten it?" }}</a>
static/templates/settings/api_key_modal.hbs:                <a id="download_zuliprc" download="{{zuliprc}}" class="button sea-green">
static/templates/settings/organization_profile_admin.hbs:        <a href="/login/?preview=true" target="_blank" rel="noopener noreferrer" class="button rounded sea-green w-200 block" id="id_org_profile_preview">

```

Reply:
So maybe make this 3 commits:
* Fix that "forgot password" link to not mention sea-green.
* This `text-decoration-line` as a separate commit, mentioning that this fixes having an `a` element with button styles to not have underlining, which we had in a few existing buttons
* The final commit doing this button.

Reply:
okay.

Reply:
> * Fix that "forgot password" link to not mention sea-green.

what do ou mean should I remove it or make it green button.
![screencapture-localhost-9991-1617699843606](https://user-images.githubusercontent.com/56171689/113686811-4db02a00-96e5-11eb-825a-2d6f9fd91454.png)

Reply:
I have also removed the underline from "forgotten it" link in the first commit.